### DISCUSSION TITLE
Please push back against the "stealing" mime

### DISCUSSION THREAD

**Discussion by PaoloTCS** [community/community#20156] [Copilot Conversations]
There has been a lot of bad PR over GitHub Copilot "stealing" code.  The general impression is that Copilot will actually write code for you, which we know is not true, one needs to carefully inspect it, and very often modify it considerably.  It writes A.I. assisted snippets, boilerplate, and it is a wonderful teaching aid, but a far cry from having a programmer at your side.  The whole thing is part of the madness that believes the hype of how far we are in achieving "Artificial Intelligence".  I am happy to pay $100/year for the advanced snippets help. and tat is perhaps how it could be marketed.  
Regards,
Paolo

**Comment by D7EAD** [comment]
Yes, frankly, Github Copilot is called Copilot for a reason. It is certainly a far cry from a pilot programmer and very useful for getting larger codebases started--but don't expect it to do the work for you!
